### PR TITLE
Split Data Module - remote and repository

### PR DESCRIPTION
**Pull request by hossein69ameri**
Create remote and repository Module into Data Module

### CODE REVIEW

File: data/remote/build.gradle.kts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+}
+
+dependencies {
+
+    implementation(libs.androidx.core.ktx)
+    implementation(project(":domain"))
```
Comment (Kaaveh) [inline data/remote/build.gradle.kts]: Don't depend this module on the domain module.

File: data/remote/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest />
```
Comment (Kaaveh) [inline data/remote/src/main/AndroidManifest.xml]: Delete this file.

File: data/remote/src/main/java/com/ameri/remote/network/ApiServices.kt [hunk 3]
```diff
@@ -1,8 +1,8 @@
-package com.ameri.data.network
+package com.ameri.remote.network
 
 
-import com.ameri.data.*
 import com.ameri.domain.models.ResponseData
```
Comment (Kaaveh) [inline data/remote/src/main/java/com/ameri/remote/network/ApiServices.kt]: Move `ResponseData` in remote module

File: data/repository/src/main/AndroidManifest.xml [hunk 4]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<manifest />
```
Comment (Kaaveh) [inline data/repository/src/main/AndroidManifest.xml]: delete this file

File: settings.gradle.kts [hunk 5]
```diff
@@ -23,4 +23,6 @@ rootProject.name = "NewsApi"
 include(":app")
 include(":presentation")
 include(":domain")
-include(":data")
\ No newline at end of file
+include(":data")
```
Comment (Kaaveh) [inline settings.gradle.kts]: remove this line

**Review by Kaaveh — CHANGES_REQUESTED** [review]

**Comment by hossein69ameri** [comment]
Delete Unused Files , Fix Gradle , Move Dto to Remote Module

**Comment by hossein69ameri** [comment]
Delete gitignore

**Review by Kaaveh — APPROVED** [review]

**Pull request merged**